src/xbps/index.md

```diff
@@ -30,10 +30,14 @@ Most general package management is done with the following commands:
 Most questions can be answered by consulting the man pages for these tools,
 together with the [xbps.d(5)](https://man.voidlinux.org/xbps.d.5) man page.
 
-To learn how to build packages from source, refer to [the README for the
+To learn how to build packages from source, refer to [the README of the
 void-packages
 repository](https://github.com/void-linux/void-packages/blob/master/README.md).
 
+To learn how to create new package templates, refer to [the Manual of the
+void-packages
+repository](https://github.com/void-linux/void-packages/blob/master/Manual.md).
```

Comment:
Can we phrase this in a different way from the previous paragraph? Just copying it and changing a few words doesn't make it readable. It could be something like "..., Manual.md in the same repository describes how to create, update or modify packages."

Reply:
Generally, of course yes. But I actually find it pretty readable this way. I did not just copy it because I'm lazy, I just wanted to keep it as simple as possible. IMO, the repetition kind of emphasizes the difference between *building* packages and *creating* package templates. Your version right now says "create, update or modify packages", but "create packages" could be misunderstood for "building packages" by beginners, as @abenson mentioned. I would stick with "package templates" in the second sentence. I would also omit the "update" in your version, since that *could* still be misunderstood for updating the packages on the system, and also it's included in "modifying packages" IMO.

Reply:
Maybe a broader rewrite will better clarify the distinction:
> The [README] provides a general overview of xbps-src and instructions for building packages locally from existing templates. For information about adding new templates or altering existing templates, refer to the [Manual].